runtime/cudaq/platform/default/rest/RemoteRESTQPU.cpp

```diff
@@ -211,15 +208,15 @@ class RemoteRESTQPU : public cudaq::QPU {
 
     // Loop through the file, extract the pass pipeline and CODEGEN Type
     auto lines = cudaq::split(configContents, '\n');
-    for (auto &line : lines) {
-      if (line.find(platformLoweringConfig) != std::string::npos) {
-        auto keyVal = cudaq::split(line, '=');
-        auto value = std::regex_replace(keyVal[1], std::regex("\""), "");
-        cudaq::info("Appending lowering pipeline: {}", value);
-        passPipelineConfig += "," + value;
-      } else if (line.find(codeEmissionType) != std::string::npos) {
-        auto keyVal = cudaq::split(line, '=');
-        codegenTranslation = keyVal[1];
+    std::regex pipeline("PLATFORM_LOWERING_CONFIG\\s*=\\s*\"(\\S+)\"");
```

Comment:
```suggestion
    std::regex pipeline("PLATFORM_LOWERING_CONFIG\\s*=\\s*\"([^\"]*)\"");
```
Since this is quoted, it seems like it could be empty or contain whitespace.

Reply:
Yes, but in either case there would be nothing to add to the pipeline, so I failure to match has the same effect.